File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/While/lista_fabio/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q10.js
import {question} from "readline-sync"

function main(){
    //input
    const num1 = parseInt(question("Enter a number: "))
    const num2 = parseInt(question("Enter a second number: "))

    //Processing
    const quotient = Math.floor(num1 / num2)
    const remainder = num1 % num2  

    //Output
    console.log(`The quotient and the remainder when dividing ${num1} by ${num2} are respectively: ${quotient} and ${remainder}`)

}

main()

File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q10.py
def main():
    #Input
    num1 = int(input("Enter a number: "))
    num2 = int(input("Enter a second number: "))

    #Processing
    quotient = round(num1 / num2)
    remainder = num1 % num2

    #Output
    print(f"The quotient and the remainder when dividing {num1} by {num2} are respectively: {quotient} and {remainder}")

main()
